src/System.IO.Compression/src/System/IO/Compression/GZipStream.cs

```diff
@@ -10,42 +10,26 @@ namespace System.IO.Compression
     public class GZipStream : Stream
     {
         private DeflateStream _deflateStream;
+        private const int _gzipWindowBits = 31;
```

Comment:
Nit: Should be ```GZipWindowsBits```.  I think it'd also be worth a comment explaining that this value is what causes zlib to write/expect a gzip header/footer, e.g. it's 15 + 16 where 16 is the "use gzip" magic number.  I had to go read the docs to understand why your change was working correctly.